tests/system/lab/setup

```diff
@@ -0,0 +1,532 @@
+#!/bin/bash
+
+# FogLAMP IP Address
+IP='192.168.4.45'
+# PI Server IP Address
+PI_IP='192.168.4.41'
+#PI Server Producer Token
+TOKEN='uid=x'
+# Max times to retry checking something before giving up and declaring a failed run
+MAX_RETRIES=100
+# Time to sleep to fix bugs. This should be zero.
+SLEEP_FIX=10
```

Comment:
Make all these args, and CI job should be able to pass these to entry point script